protos/feast/core/CoreService.proto

```diff
@@ -225,7 +227,7 @@ message ListProjectsResponse {
 message GetFeatureStatisticsRequest {
     // Optional filter which filters returned statistics by selected features. These
     // features must be present in the data that is being processed.
-    repeated string feature_refs = 1;
+    repeated string feature_ids = 1;
```

Comment:
This should be feature refs right?

Reply:
No, feature refs should refer to the FeatureRef object for consistency